pkg/apis/pingcap/v1alpha1/defaulting/tidbcluster.go

```diff
@@ -60,6 +60,10 @@ func setTidbClusterSpecDefault(tc *v1alpha1.TidbCluster) {
 		d := false
 		tc.Spec.EnablePVReclaim = &d
 	}
+	recyclePVP := corev1.PersistentVolumeReclaimRecycle
```

Comment:
Suggest using retain as the default policy. WDYT @cofyc 
Retain is what we suggested in our docs and only few PVs support recycle.

Reply:
updated.